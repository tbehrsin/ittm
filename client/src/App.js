import React from 'react';
import { Users } from 'components/Users';

import styles from './App.module.scss';
import classNames from 'classnames/bind';

const cx = classNames.bind(styles);

function App() {
  return (
    <div className={cx('app')}>
      <Users />
    </div>
  );
}

export default App;
