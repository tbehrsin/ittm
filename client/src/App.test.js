import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

jest.mock('components/Users', () => ({
  __esModule: true,
  Users: () => {
    return 'Users';
  },
}));

test('renders learn react link', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
