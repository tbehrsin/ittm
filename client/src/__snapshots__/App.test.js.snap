// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    Users
  </div>
</DocumentFragment>
`;
